site/src/pages/docs/client-http.mdx

```diff
@@ -23,6 +25,43 @@ AggregatedHttpResponse res3 = webClient.post("/upload", "{ \"foo\": \"bar\" }")
                                        .aggregate().join();
 ```
 
+You can also create <type://WebClient> without a base URI and send a request with an absolute URI.
+The ad-hoc <type://WebClient> would be useful in the following cases:
+
+- Send requests to arbitrary endpoints
+- Build a proxy server
+- Handle redirected requests
+
+```java
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.HttpHeaderNames;
+
+// Create a WebClient without a base URI.
+WebClient webClient = WebClient.of();
+
+// Send a request with an absolute URI.
+AggregatedHttpResponse res1 = webClient.get("http://example.com/foo/bar.txt")
+                                       .aggregate()
+                                       .join();
```

Comment:
@tobias- told me this did not work for him. Could you double-check if this example works as expected?

Reply:
Sure! Let me check it.

Reply:
Tested and it worked as expected. 😀
Currently ad-hoc `WebClient` is verified by:
https://github.com/line/armeria/blob/fbe90f0f2a952578012a552c26b3f4f8bdb33c8c/core/src/test/java/com/linecorp/armeria/client/HttpClientRequestPathTest.java#L107-L118